src/components/specific/models/models-manager/dwg-manager/DWGManager.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <GenericModelsManager :project="project" :tabs="tabs" />
+  <GenericModelsManager :project="project" :type="type" :tabs="tabs" />
```

Comment:
It looks like you forgot to declare/add the `file-uploaded` event on this component.
The same is true for `PDFManager` and `PointCloudManager`.
This will work however, for the same reason as for the `type` prop as we discuss yesterday.
It would be more explicit to forward the event anyway though.